src/features/behandling/behandlingUtils.ts

```diff
@@ -24,3 +31,16 @@ export function erAvslått(behandling: Behandling): boolean {
         Behandlingsstatus.BEREGNET_AVSLAG,
     ].some((status) => behandling.status === status);
 }
+
+export const hentSisteVurderteVilkår = (behandlingsinformasjon: Behandlingsinformasjon) => {
+    const vilkår = mapToVilkårsinformasjon(behandlingsinformasjon);
+    const lastElement = pipe(
+        vilkår,
+        findLast((v: Vilkårsinformasjon) => v.erStartet)
+    );
+
+    if (isNone(lastElement)) {
+        return Vilkårtype.Uførhet;
+    }
+    return lastElement.value.vilkårtype;
```

Comment:
Legg til
```ts
import * as Array from 'fp-ts/Array';
import * as Option from 'fp-ts/Option';
```

så kan vi gjøre

```suggestion
    const vilkår = mapToVilkårsinformasjon(behandlingsinformasjon);
    const lastElement = pipe(
        behandlingsinformasjon,
        mapToVilkårsinformasjon,
        Array.findLast((v: Vilkårsinformasjon) => v.erStartet),
        Option.fold(
            () => Vilkårtype.Uførhet,
            x => x.vilkårtype
        )
    );
```